main.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+function createTree(container, data) {
+  const ul = document.createElement('ul');
+  container.append(ul);
```

Comment:
will be better to append to existing DOM element when you completely create it with children. It will be more performant